src/i18n/es-es/index.js

```diff
@@ -191,6 +191,8 @@ export default {
         connect_wallet: 'Conectar billetera',
         view_address: 'Ver dirección',
         disconnect: 'Desconectar',
+        disable_wallet_extensions: 'Desactiva los monederos de extensión o establece Brave Wallet como predeterminado en la configuración del monedero del navegador para utilizar Brave Wallet.',
+        enable_wallet_extensions: 'Habilitar la extensión MetaMask y establecer por defecto `preferir extensiones` en la configuración del monedero del navegador para utilizar el monedero MetaMask.',
```

Comment:
These two should be added also to the Portuguese translation PR.

Reply:
@karynemayer is adding this.